src/interfaces/IHook.sol

```diff
@@ -2,14 +2,10 @@
 pragma solidity ^0.8.0;
 
 import {PackedUserOperation} from "account-abstraction/interfaces/PackedUserOperation.sol";
+import {IERC1271} from "./IERC1271.sol";
```

Comment:
hm so a hook doesnt need to have any of these interfaces tho

Reply:
i guess if we want this to define every possible entrypoint it should also have the before/after execute? is that in a sep pr?